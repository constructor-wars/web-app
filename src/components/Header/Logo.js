import React from 'react';
import Logoimg from '../../images/constructor-labs-badge.png';

export default function Logo() {
  return (
    <div>
      <img className="logoimg" src={Logoimg} />
    </div>
  );
}
