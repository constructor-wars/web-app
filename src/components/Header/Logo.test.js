import React from "react";
import Logo from "./Logo";
import renderer from "react-test-renderer";

describe("Logo", () => {
  it("matches the snapshot", () => {
    const tree = renderer.create(<Logo />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
