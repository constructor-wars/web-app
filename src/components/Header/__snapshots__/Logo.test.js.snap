// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo matches the snapshot 1`] = `
<div>
  <a
    href="/"
  >
    <img
      className="logoimg"
      src="test-file-stub"
    />
  </a>
</div>
`;
